code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -39,7 +39,7 @@
 	name = "antique laser gun"
 	icon_state = "caplaser"
 	w_class = WEIGHT_CLASS_NORMAL
-	inhand_icon_state = null
+	inhand_icon_state = "laser"
 	desc = "This is an antique laser gun. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The object menaces with spikes of energy. On the item is an image of Space Station 13. The station is exploding."
```

Comment:
This has a unique inhand sprite. I believe the reason this is null has to do with how shaded charge works as well, but this straight up will break the guns inhand sprites regardless.

Reply:
let me test this ingame real quick: i genuinely thought it just defaulted to "laser", this may be my bad